src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -34,11 +34,11 @@
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
     // Register this class as a VS package.
-    [PackageRegistration(UseManagedResourcesOnly = true)]
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
```

Comment:
This change follows the same pattern that Mads Kristensen used for converting the SonarLintDaemon package [here](https://github.com/SonarSource/sonarlint-visualstudio/commit/fd2b0ab5b675faab60d2b36dc0d0ed82100f86d5). Basically, we queue up some work to be done on the UI thread later.

Components that use MEF or that call _IServiceProvider.GetService_ can't be initialised on a background thread. We could convert some of the components so they can be initialised in the background, but that would required more work.